app/tsconfig.json

```diff
@@ -23,7 +23,8 @@
     "frontend/**/*.js",
     "frontend/**/*.svelte",
     "logic/util/Observable.ts",
-    "logic/build.ts"
+    "logic/build.ts", 
+    "lingui.config.ts"  
```

Comment:
Please avoid changing the last line. Could you please add the new file after `Observable.ts`?